README.md

```diff
@@ -12,7 +12,6 @@ This action enables the integration of PagerDuty into StackStorm. It is capable
 Copy the example configuration in [pagerduty.yaml.example](./pagerduty.yaml.example)
 to `/opt/stackstorm/configs/pagerduty.yaml` and edit as required.
 
-* `subdomain:` name of subdomain
 * `api_key:` API-KEY
 * `service_api:` SERVICE-API
 * `debug:` optional debug flag. Set to True for additional logging
```

Comment:
Can you also add a line here along the lines of "after editing the configuration, run `sudo st2ctl reload --register-configs` to ensure your configuration is loaded in StackStorm?